### PR TITLE
Updated Sample app and cleaned up unit tests for AppUsers

### PR DESCRIPTION
**Pull request by achunBOX**

### CODE REVIEW

File: BoxContentSDK/BoxContentSDKTests/BOXContentClientTestCase.m [hunk 1]
```diff
@@ -7,7 +7,78 @@
...
+#import "BOXContentClient+Folder.h"
+#import "BOXContentClient_Private.h"
+#import "BOXFolderRequest.h"
+#import "BOXRequest_Private.h"
 
 @implementation BOXContentClientTestCase
 
+- (void)fetchAccessTokenWithCompletion:(void (^)(NSString *, NSDate *, NSError *))completion
+{
+    completion(@"access_token", [NSDate dateWithTimeIntervalSinceNow:100], nil);
```
Comment (RicoYao) [inline BoxContentSDK/BoxContentSDKTests/BOXContentClientTestCase.m]: Can you make this a #define or property or whatever.

File: BoxContentSDK/BoxContentSDKTests/BOXContentClientTestCase.m [hunk 2]
```diff
@@ -7,7 +7,78 @@
 //
 
 #import "BOXContentClientTestCase.h"
+#import "BOXContentClient.h"
+#import "BOXAPIAccessTokenDelegate.h"
+#import "BOXContentClient+Folder.h"
+#import "BOXContentClient_Private.h"
+#import "BOXFolderRequest.h"
+#import "BOXRequest_Private.h"
```
Comment (RicoYao) [inline BoxContentSDK/BoxContentSDKTests/BOXContentClientTestCase.m:23]: Can you add a test to enforce what the expected behavior is if fetchAccessToken returns a nil access token?
